src/native/windows/src/service.c

```diff
@@ -575,6 +575,57 @@ apxServiceControl(APXHANDLE hService, DWORD dwControl, UINT uMsg,
     return FALSE;
 }
 
+BOOL
```

Comment:
A comment here would be better. Think of folks reading the code in the future ;-)

Reply:
There are no comments in the whole file I would rather make another commit explaining each routine in the include.

Reply:
Nope, if YOU add a function/method, then YOU should document. Then, if you feel like documenting existing code, you can do that as well.